engine/lib/views.php

```diff
@@ -1428,6 +1429,25 @@ function elgg_view_input($input_type, array $vars = array()) {
 }
 
 /**
+ * Renders a form field
+ * A shortcut method to rendering inputs from an array of options
+ * In addition to all $vars accepted by elgg_view_input(), this function
+ * offers support for:
+ *  - #type: input type
+ *
+ * @see elgg_view_input()
+ *
+ * @param array $vars Field vars
+ * @return string
+ */
+function elgg_view_field(array $vars = []) {
+	$input_type = elgg_extract('#type', $vars);
```

Comment:
I am thinking in the future, we will follow this Drupaley approach to decorating fields with validation rules, various setter/getter/sanitizer callbacks.